app/src/main/kotlin/com/simplemobiletools/calendar/pro/fragments/WeekFragmentsHolder.kt

```diff
@@ -49,7 +54,7 @@ class WeekFragmentsHolder : MyFragmentHolder(), WeekFragmentListener {
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
         weekHolder = inflater.inflate(R.layout.fragment_week_holder, container, false) as ViewGroup
         weekHolder!!.background = ColorDrawable(context!!.config.backgroundColor)
-
+        mConfig = requireContext().config
```

Comment:
do not store it here, just in onResume. Also rename "customise" to "customize"

Reply:
Do you mean to move initialisation to `onResume` ?

Reply:
Like this
```
    override fun onResume() {
        super.onResume()
        val allowCustomizeDayCount = requireContext().config.allowCustomizeDayCount
        week_view_seekbar.beVisibleIf(allowCustomizeDayCount)
        week_view_days_count.beVisibleIf(allowCustomizeDayCount)
    }
```

Reply:
ye maybe, could be

Reply:
or actually handle null context too, anything can happen in onResume

Reply:
Okay. Will do.